### PR TITLE
Release/v.2.0.2

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- #459 
- #460 

# Review By (Date)
- Soonish?

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code and confirm what has been changed and reviewed.

### CODE REVIEW

File: src/components/composite/itineraryItem.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const ItineraryItem = ({ blok }) => {
     image: { filename, focus } = {},
     caption,
   } = blok;
-  // First take all but the last selected meal options and joins them together with a comma.
+  // First take all but the last selected meal options and joins them together with a comma
```
Comment (yvonnetangsu) [inline src/components/composite/itineraryItem.js:25]: This line was just me trying to jiggle the github action to restart (somehow clicking rerun action didn't do it)

**Comment by yvonnetangsu** [comment]
@sherakama I redeployed on Netlify and everything seems to be working, it just wouldn't trigger the GitHub action test to refresh themselves 🤔 
https://app.netlify.com/sites/stanford-alumni/deploys
Screen Shot 2022-09-12 at 9 58 40 AM

File: src/components/composite/itineraryItem.js (hunk 1 above)
Comment (yvonnetangsu) [reply src/components/composite/itineraryItem.js]: I just undid this change so I can push up a commit to jiggle the github actions in this PR

**Review by sherakama — APPROVED** [review]
GTG. TY Yvonne.

**Pull request merged**